testcases/misc_tests/cca_export_import_test.c

```diff
@@ -0,0 +1,1448 @@
+/*
+ * COPYRIGHT (c) International Business Machines Corp. 2020
+ *
+ * This program is provided under the terms of the Common Public License,
+ * version 1.0 (CPL-1.0). Any use, reproduction or distribution for this
+ * software constitutes recipient's acceptance of CPL-1.0 terms which can
+ * be found in the file LICENSE file or at
+ * https://opensource.org/licenses/cpl1.0.php
+ *
+ * import/export test for the CCA token
+ * by Harald Freudenberger <freude@de.ibm.com>
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <memory.h>
+#include <stdint.h>
+
+#include "pkcs11types.h"
+#include "ec_curves.h"
+
+#include "regress.h"
+#include "common.c"
+
+// define this to enable the AES cipher key import test:
+// #define CCA_AES_CIPHER_KEY_SUPPORTED
```

Comment:
I don't find this define anywhere else except in this testcase, especially not in cca_specific.c. Is this a left over, or did you leave it there intentionally?

Reply:
You are right. This is local to the file.
The code to support import of CCA AES cipher blobs is easy - just add it to the switch within the symetric key import code function. However, I did not want to discard the test code as I spend some time to develop it and so I commented it out with this #ifdef CCA_AES_CIPHER_KEY_SUPPORTED